modules/rntuple.mjs

```diff
@@ -110,6 +110,64 @@ class RBufferReader {
 
 }
 
+const ENTupleColumnType = {
+  kUnknown: 0,
+  kIndex64: 1,
+  kIndex32: 2,
+  kSwitch: 3,
+  kByte: 4,
+  kChar: 5,
+  kBit: 6,
+  kReal64: 13,
```

Comment:
Please write these numbers in hexadecimal so they are more easily comparable with the BinarySpecification and the C++ version